recipes/tensorflow-probability/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "tensorflow-probability" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/probability/archive/v{{ version }}.tar.gz
+  sha256: 6e0e21b40555cbc1725f2fde96fa7705c160d856ae67f84107ad889896314694
+  patches:
+    - 0001-always-build-release.patch
+
+build:
+  number: 0
+  script: 
+    - rm -f BUILD
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [win or osx or py>36]
```

Comment:
We can merge this as is. It would be great to build this on the other platforms though.